Android/android.py

```diff
@@ -671,6 +677,14 @@ def package(context):
                     else:
                         shutil.copy2(src, dst, follow_symlinks=False)
 
+        # Strip debug information.
+        if not context.debug:
+            run(
+                [android_env(context.host)["STRIP"]]
+                + glob(f"{temp_dir}/**/*.so", recursive=True),
```

Comment:
Consider unpacking instead of concut?
```suggestion
                [
                    android_env(context.host)["STRIP"],
                    *glob(f"{temp_dir}/**/*.so", recursive=True),
                ],
```